fpga_interchange/examples/chipdb.cmake

```diff
@@ -56,6 +56,58 @@ function(create_rapidwright_device_db)
     endif()
 endfunction()
 
+function(create_prjoxide_device_db)
```

Comment:
Might be worth having each FPGA family-specific functions in separate files, as the `cipdb.cmake` file could potentially get quite big in the future.

Reply:
Yes, that seems sensible